mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -185,19 +212,26 @@ def _SendConfiguration(self, configuration, fireandforget=False, timeout=2.0, ch
     #
 
     def StartObjectDetectionTask(self, taskId=None, systemState=None, visionTaskParameters=None, timeout=2.0, **ignoredArgs):
-        # type: (Optional[types.SystemState], Optional[types.visionTaskObjectDetectionParametersSchema], float, Dict) -> Optional[Dict]
+        # type: (Optional[str], Optional[Union[types.SystemState, Dict]], Optional[Union[types.visionTaskObjectDetectionParametersSchema, Dict]], float, Any) -> Optional[Dict[str, str]]
         """Starts detection thread to continuously detect objects. the vision server will send detection results directly to mujin controller.
 
         Args:
-            taskId (str, optional): If specified, the specific taskId to use
-            systemState (types.SystemState or dict): The state of the system. Used to select the profile that the vision task will use. See "Profile Selection" documentation for more details.
-            visionTaskParameters (types.visionTaskObjectDetectionParametersSchema or dict): Parameters for the object detection task. These take precedence over the base profile selected via the system state, but are overwritten by the overwrite profile.
+            taskId (str, optional): If specified, the specific taskId to use.
+            systemState (types.SystemState | dict, optional): The state of the system. Used to select the profile that the vision task will use. See "Profile Selection" documentation for more details.
+            visionTaskParameters (types.visionTaskObjectDetectionParametersSchema | dict, optional): Parameters for the object detection task. These take precedence over the base profile selected via the system state, but are overwritten by the overwrite profile.
+            timeout (float, optional): Time in seconds after which the command is assumed to have failed. (Default: 2.0)
 
         Returns:
             dict: Returns immediately once the call completes
+
+                A dictionary with the structure:
+
+                - taskId (str): The taskId of the created task
         """
         log.verbose('Starting detection thread...')
-        command = {'command': 'StartObjectDetectionTask'}  # type: Dict[str, Any]
+        command = {
+            'command': 'StartObjectDetectionTask',
+        }  # type: Dict[str, Any]
```

Comment:
I think the type checker expects this to be on the first line

Reply:
No, at least Pylance expects the comment here. It shows an error if I put the comment on the first line.